### PR TITLE
Install sample directory

### PR DESCRIPTION
**Pull request by jmespadero**
Add rules to copy meshes in the src/distrib/sample to BUILD_DIR/sample and INSTALL_DIR/shared/meshlab/sample

@rpavlik please revise and feel free to discuss anything.

### CODE REVIEW

**Comment by jmespadero** [comment]
Due to its size, I have added two separate install options for directory sample

* option(INSTALL_SAMPLE_MESHES    "Should the sample meshes in src/distrib/samples be installed? It will increase the build size in 14 Mb."     ON)
* option(INSTALL_SAMPLE_RANGEMAPS    "Should the sample rangemaps in src/distrib/samples/rangemaps be installed? It will increase the build size in 67 Mb."     OFF)

**Comment by jmespadero** [comment]
Plan C is don't include samples directory and add a new menu option `Help` ->`Download samples` with a link to a repository (in github) where user can download meshes and tutorials after the installation.

**Comment by rpavlik** [comment]
Seems reasonable, though I would have done it more like how I did the shaders.  I skipped installing the samples because the Debian package skipped them due to the unclear copyright/license status of them. Additionally, I'm not sure they actually need to be copied to the build tree, since if you have the build tree you probably have the source tree, and as far as I know e.g. they aren't required to be in any particular location with respect to the executable to be found, unlike the shaders and the plugin xml files.

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -321,6 +331,28 @@ foreach(FN ${SHADERS})
...
+        
+    #Rules to install sample files from distrib/sample directory
+    install(
+        DIRECTORY ${MESHLAB_SAMPLE_OUTPUT_DIR}
+        DESTINATION ${MESHLAB_SAMPLE_INSTALL_DIR}
+        COMPONENT Samples)
+    file(GLOB_RECURSE SAMPLE 
+         LIST_DIRECTORIES false
+         ${MESHLAB_SAMPLE_OUTPUT_DIR}/* )
+    list(APPEND COPIED_FILES ${SAMPLE} )
```
Comment (rpavlik) [inline src/CMakeLists.txt]: this isn't what the copied_files thing is for - it's for files that have received an `add_custom_command` to copy. and thus need a target to mention them to perform the copy.

**Comment by jmespadero** [comment]
Ok, so I understand that I should close this PR. Correct?

Also, I understand the desition of Debian and I agree with that. So I would start to prepare a set of tutorials (not just sample files) with exercises like "smooth this noisy mesh", "close holes smallers than this threshold", "convert vertex texture coordinates to wedge coordinates", etc..

**Comment by rpavlik** [comment]
If you like #538 and think it's better than this one, you can close it, yes.

**Comment by jmespadero** [comment]
I prefer your version. Thanks for your changes.

**Pull request closed without merging**